source/troubleshooting.txt

```diff
@@ -166,9 +166,9 @@ frameworks.
    if __name__ == "__main__":
        app.run()
 
-.. tip::
+.. TODO: Add to retrieve page .. tip::
```

Comment:
Commented this out to get rid of a build error

Reply:
Yeah, I deleted that line in my recent PR. I couldn't find that ref or one resembling it.